sdk/nodejs/src/client.ts

```diff
@@ -206,6 +202,7 @@ export class DevCycleClient<
         this.openFeatureProvider = new DevCycleProviderClass(this, {
             logger: this.logger,
         })
+        this.setPlatformDataInBucketingLib()
```

Comment:
is this data now only being set for openFeatureProvider?

Reply:
its also set at line 135 above